deploy/ansible/ansible.cfg

```diff
@@ -131,8 +131,8 @@ become_ask_pass = True
 # SSH settings
 ########################################
 
-#private_key_file	= ...
-host_key_checking	= True
+private_key_file	= ~/.ssh/sorakey
```

Comment:
Why do we specify this exact key? :)

Reply:
@neewy this is just the filename for convenience, if you don't want to specify `--private-key=<key>` in the CLI during `ansible-playbook` launch, you can save your private key with this name, or change this name.

Reply:
ok good